src/Umbraco.Tests/IO/PhysicalFileSystemTests.cs

```diff
@@ -13,8 +14,8 @@ namespace Umbraco.Tests.IO
     public class PhysicalFileSystemTests : AbstractFileSystemTests
     {
         public PhysicalFileSystemTests()
-            : base(new PhysicalFileSystem(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "FileSysTests"),
-                "/Media/", new IOHelper()))
+            : base(new PhysicalFileSystem(IOHelper.Default, Current.Logger, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "FileSysTests"),
```

Comment:
Not sure how Current.Logger is working here since AbstractFileSystemTests doesn't set current